packages/common/container-definitions/src/fluidPackage.ts

```diff
@@ -7,6 +7,7 @@
  * Specifies an environment on Fluid property of a IFluidPackage.
  * @alpha
  */
+// eslint-disable-next-line @typescript-eslint/consistent-indexed-object-style
```

Comment:
disabled so the API doesn't change - not sure there's any difference though.

Reply:
There seem to be enough of these that it seems like step 1 is to use strict but roll this one back.
A lot of single line disables is a could be preferred or not. In the case of not really knowing, I like not single lines.

Another good rule is that all disables have a comment, saying why.
I didn't look this one rule up in particular. Just commenting on the general pattern.